pro/src/components/pages/Offers/Offer/OfferDetails/OfferForm/OfferForm.jsx

```diff
@@ -296,7 +296,10 @@ const OfferForm = ({
           ...oldFormErrors,
           venueId: 'Il faut obligatoirement une structure avec un lieu.',
         }))
-      } else {
+      } else if (
+        formErrors['venueId'] ===
+        'Il faut obligatoirement une structure avec un lieu.'
+      ) {
```

Comment:
Ca reste du hack mais je pense qu'il faudrait plutot tester que venueId n'est vide avant de vider les erreurs.

```suggestion
      } else if (formValues.venueId !== DEFAULT_FORM_VALUES['venueId']) {
    //...
    , [offerSubCategory, setOfferVenue, venues, formValues.venueId]
```